### PR TITLE
fix: fixing the type error when activating the 'collect data' modal and refactor

### PR DESCRIPTION
**Pull request by skdud4659**
### To Reviewers
- [ ] Skip (`style`, `chore`, `ci` ONLY)
- [ ] Not that difficult

### Type of Change
- [ ] New feature
- [x] Bug fixes
- [ ] Feature improvement
- [x] Refactor
- [ ] Others (performance improvement, CI/CD, etc.)

### Affects to
- [ ] Packages
  - [ ] core-lib
  - [ ] mirinae
  - [ ] etc 

- [ ] Apps
  - [ ] storybook
  - [x] web

### Checklist
- Did you check the `lint` and `type`?
- Did you document the changes?
  - Changes in `mirinae` should be reflected in `storybook`.
- Did you test the app after the package changes?

### Description
SSIA

### CODE REVIEW

File: apps/web/src/services/asset-inventory/collector/shared/collector-data-modal/modules/CollectorDataDuplicationInner.vue [hunk 1]
```diff
@@ -70,19 +70,23 @@ import { PDefinitionTable, PI, PStatus } from '@spaceone/design-system';
...
 
+import type { CollectorData } from '@/services/asset-inventory/collector/shared/collector-data-modal/type';
 import { ACCOUNT_TYPE } from '@/services/asset-inventory/collector/shared/collector-data-modal/type';
 
 const SUCCEEDED_COLOR = green[400];
 const FAILED_COLOR = coral[400];
 
 interface Props {
     accountType?: string;
+    item?: CollectorData;
```
Comment (yuda110) [inline apps/web/src/services/asset-inventory/collector/shared/collector-data-modal/modules/CollectorDataDuplicationInner.vue:81]: question: Why is this needed? It doesn't seem to be used anywhere.🧐

Comment (yuda110) [reply apps/web/src/services/asset-inventory/collector/shared/collector-data-modal/modules/CollectorDataDuplicationInner.vue:81]: Oh, it seems like it will be used later when real data comes in.

Comment (skdud4659) [reply apps/web/src/services/asset-inventory/collector/shared/collector-data-modal/modules/CollectorDataDuplicationInner.vue:81]: It will probably be necessary when connecting real data!

**Review by yuda110 — APPROVED** [review]
LGTM!!!
